iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift

```diff
@@ -202,6 +203,10 @@ class PlacePageInfoViewController: UIViewController {
       wheelchairView = createInfoItem(wheelchair, icon: UIImage(named: "ic_placepage_wheelchair"))
     }
     
+    if let driveThrough = placePageInfoData.driveThrough {
+      driveThroughView = createInfoItem(driveThrough, icon: UIImage(named: "ic_placepage_drive_through"))
+    }
```

Comment:
To clarify, not iOS shows drive_through=yes/no, while Android shows only drive_through=yes ?

Reply:
No. Its only showing `drive_through=yes` on both platforms